src/priority_queue.js

```diff
@@ -0,0 +1,109 @@
+'use strict'
+
+class Node {
+  constructor(data, priority, next) {
+    this.data = data
+    this.priority = priority
+    this.next = next
+  }
+}
+
+export default class Queue {
+  constructor() {
+    this.thequeue = null
+    this.size = 0
+  }
+
+  enqueue(input, priority) {
+    if (!this.thequeue) {
+      this.thequeue = new Node(input, priority, null)
+      this.size++
+      return
```

Comment:
I'm trying to end the function completely so the rest doesn't run. There's nothing to really return since the function just alters the queue